src/variables/components/VariableDropdownReactSelect.tsx

```diff
@@ -0,0 +1,369 @@
+    const {selectedValue, name, status} = this.props
+    const {shownValues} = this.state
+
+    const isDisabled = !shownValues || shownValues.length === 0
+
+    let realVals = []
+
+    //  make them all look like: { value: 'vanilla', label: 'Vanilla' }
+    if (shownValues && shownValues.length) {
+      realVals = shownValues.map(val => ({value: val, label: val}))
+    }
+    const placeHolderText = this.getPlaceHolderText('Select a Value')
+
+    const foreground = 'white'
+    const inputBackground = 'black'
+    const background = '#2c2d35'
+    const focusColor = '#46454e'
+    const selectColor = '#ce58eb'
+
+    const customStyles = {
+      option: (provided, state) => {
+        let backgroundColor = state.isFocused ? focusColor : background
+
+        if (state.isSelected) {
+          backgroundColor = selectColor
+        }
+
+        return {
+          ...provided,
+          color: foreground,
+          backgroundColor,
+          padding: 10,
+        }
+      },
+      clearIndicator: provided => {
+        // console.log("clear indicator...state??? ACK", state)
+        // const foregroundColor = state.isFocused? 'yellow' : 'cyan'
+        return {
+          ...provided,
+          '&:hover': {
+            color: 'white',
+          },
+        }
+      },
+      control: () => ({
+        width: 225,
+        height: 40,
+        color: foreground,
+        padding: 3,
+        backgroundColor: background,
+        display: 'flex',
+      }),
+      valueContainer: provided => ({
+        ...provided,
+        color: foreground,
+        backgroundColor: inputBackground,
+      }),
+      placeHolder: provided => ({
+        ...provided,
+        color: foreground,
+        backgroundColor: inputBackground,
+      }),
+      indicatorSeparator: () => ({
+        display: 'none',
+      }),
+      dropdownIndicator: (provided, state) => {
+        const hoverBackground = state.isFocused ? '#5d5f6f' : '#828497'
+
+        return {
+          ...provided,
+          color: foreground,
+          '&:hover': {
+            background: hoverBackground,
+            color: foreground,
+          },
+        }
+      },
+      singleValue: (provided, state) => {
+        const opacity = state.isDisabled ? 0.5 : 1
+        const transition = 'opacity 300ms'
+
+        return {
+          ...provided,
+          opacity,
+          transition,
+          color: foreground,
+          backgroundColor: inputBackground,
+        }
+      },
+      menu: provided => ({
+        ...provided,
+        zIndex: 15,
+        backgroundColor: background,
+      }),
+    }
+
+    const onChange = selection => {
+      if (selection && selection.value) {
+        this.handleSelect(selection.value)
+      }
+    }
+    // primary50 is the color that is used when an item is clicked on to change the selection
+    // have not found an alternative way (via the customStyles) to change that color)
+
+    //    const style={width:200}
```

Comment:
✂️

Reply:
i removed the // const style
line; but not the comments about primary50